hera_sim/beams.py

```diff
@@ -208,7 +611,7 @@ def interp(self, az_array, za_array, freq_array, reuse_spline=None):
 
             # Updated polar coordinates
             theta_s = np.sqrt(Xs ** 2.0 + Ys ** 2.0)
-            phi_s = np.arccos(Xs / theta_s)
+            phi_s = np.where(theta_s == 0.0, 0.0, np.arccos(Xs / theta_s))
```

Comment:
This is still going to throw up annoying warnings of divide by zero. Can you make it two lines and actually mask the calculation?